.docker/Dockerfile.solr

```diff
@@ -1,18 +1,21 @@
 ##
 # @see  https://govcms.gov.au/wiki-advanced#docker
 #
+FROM amazeeio/solr:7.5-drupal
```

Comment:
Note this is all aligned to what GovCMS8 SaaS is providing as a base: https://github.com/govCMS/govcms8lagoon/blob/develop/.docker/Dockerfile.solr